README.md

```diff
@@ -1,2 +1,57 @@
-# datarich-demo
-DataRich Demo
+# DataRich Documents Demo
+
+Welcome to DataRich Documents Demo!
+
+**Tutorials**:
+
+👨‍🎓 [[datarich-tutorial|Tutorial 1: Publishing DataRich Stories on DataHub]]
+
+👨‍🎓 [[frictionless-tutorial|Tutorial 2: Publishing DataRich Datasets on DataHub]]
+
+**Demo pages:**
+
+👀 [[story1|DataRich Story Example]]
+
+👀 [[story2|DataRich Dataset Example]]
+
+## But wait, what is a DataRich document?
+
+![[data-rich-document.png]]
+
+A DataRich document is one in which the writer can easily mix formatted text content with data visualisations. This means that you don't have to code or embed your charts and tables; they can be added to the document with a very simple syntax, either by passing inline data or simply referencing your data files. What you end up with is a plain text, human-readable document enriched with data visualisations, that is simple to edit and looks awesome when published with DataHub.
```

Comment:
A data rich document is just a classic document but with some superpowers to make data storytelling and analysis easy to do.